src/app/layout.tsx

```diff
@@ -3,7 +3,7 @@ import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import USABanner from './components/UsaBanner/UsaBanner';
 import './globals.scss';
-import './custom-styles.css';
+import './_styles/custom-styles.css';
```

Comment:
I'm not sure I'm seeing exactly why, but this change is breaking a lot of styling for me. Does it seem to be working for you? I tried deleting both `node_modules` and `.next` and reinstalling/re-running just to make sure paths weren't being cached or something but I'm still not having any luck.

Here's a sample of what I'm seeing:
Home
<img width="1346" alt="image" src="https://github.com/user-attachments/assets/604959c5-47bc-4523-bb12-846c8fc61332">

Case studies
<img width="1350" alt="image" src="https://github.com/user-attachments/assets/ffd0c32a-9172-437e-bd5f-71ff2b704e42">